src/main/java/org/spideruci/tacoco/MavenBuildProbe.java

```diff
@@ -31,7 +32,8 @@ public MavenBuildProbe(String absolutTargetPath) {
 	@Override
 	public List<String> getClasses() {
 		makeFilter();
-		scanner.setBasedir(targetDir+"/target/test-classes"); //MAVEN TEST CLASS FOLDER
+        String testClassDirectoryPath = StringUtils.join(new String[] {targetDir, "target", "test-classes"}, File.separator);
```

Comment:
Use a StringBuffer, Guava or write a custom utility method that does String Join